diff --git a/node_modules/@react-native-community/blur/android/build.gradle b/node_modules/@react-native-community/blur/android/build.gradle
index 8177235..0e808ab 100644
--- a/node_modules/@react-native-community/blur/android/build.gradle
+++ b/node_modules/@react-native-community/blur/android/build.gradle
@@ -8,8 +8,8 @@ buildscript {
     // module dependency in an application project.
     if (project == rootProject) {
         repositories {
+            mavenCentral()
             google()
-            jcenter()
         }
 
         dependencies {
@@ -36,12 +36,12 @@ android {
 }
 
 repositories {
+    mavenCentral()
     google()
-    jcenter()
 }
 
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.eightbitlab:blurview:1.6.3'
+    implementation 'com.eightbitlab:blurview:1.6.6'
 }
\ No newline at end of file
diff --git a/node_modules/@react-native-community/blur/src/BlurView.android.js b/node_modules/@react-native-community/blur/src/BlurView.android.js
index a192476..74e2ab9 100644
--- a/node_modules/@react-native-community/blur/src/BlurView.android.js
+++ b/node_modules/@react-native-community/blur/src/BlurView.android.js
@@ -1,10 +1,10 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 import {
   View,
   requireNativeComponent,
   DeviceEventEmitter,
-  ViewPropTypes,
   StyleSheet,
 } from 'react-native';
 
@@ -16,7 +16,7 @@ const OVERLAY_COLORS = {
 
 class BlurView extends Component {
   componentDidMount() {
-    DeviceEventEmitter.addListener('ReactNativeBlurError', message => {
+    DeviceEventEmitter.addListener('ReactNativeBlurError', (message) => {
       throw new Error(`[ReactNativeBlur]: ${message}`);
     });
   }
@@ -33,12 +33,12 @@ class BlurView extends Component {
   }
 
   blurRadius() {
-    const { blurRadius, blurAmount } = this.props;
+    const {blurRadius, blurAmount} = this.props;
 
     if (blurRadius != null) {
       if (blurRadius > 25) {
         throw new Error(
-          `[ReactNativeBlur]: blurRadius cannot be greater than 25! (was: ${blurRadius})`
+          `[ReactNativeBlur]: blurRadius cannot be greater than 25! (was: ${blurRadius})`,
         );
       }
       return blurRadius;
@@ -55,7 +55,7 @@ class BlurView extends Component {
   }
 
   downsampleFactor() {
-    const { downsampleFactor, blurRadius } = this.props;
+    const {downsampleFactor, blurRadius} = this.props;
     if (downsampleFactor != null) {
       return downsampleFactor;
     }
@@ -63,7 +63,7 @@ class BlurView extends Component {
   }
 
   render() {
-    const { style } = this.props;
+    const {style} = this.props;
 
     return (
       <NativeBlurView
@@ -71,8 +71,7 @@ class BlurView extends Component {
         downsampleFactor={this.downsampleFactor()}
         overlayColor={this.overlayColor()}
         pointerEvents="none"
-        style={StyleSheet.compose(styles.transparent, style)}
-      >
+        style={StyleSheet.compose(styles.transparent, style)}>
         {this.props.children}
       </NativeBlurView>
     );
@@ -80,7 +79,7 @@ class BlurView extends Component {
 }
 
 const styles = StyleSheet.create({
-  transparent: { backgroundColor: 'transparent' },
+  transparent: {backgroundColor: 'transparent'},
 });
 
 BlurView.propTypes = {
diff --git a/node_modules/@react-native-community/blur/src/BlurView.ios.js b/node_modules/@react-native-community/blur/src/BlurView.ios.js
index f675bf9..30ea7c1 100644
--- a/node_modules/@react-native-community/blur/src/BlurView.ios.js
+++ b/node_modules/@react-native-community/blur/src/BlurView.ios.js
@@ -1,9 +1,14 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import {View, requireNativeComponent, ViewPropTypes, StyleSheet} from 'react-native';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
+import {
+  View,
+  requireNativeComponent,
+  StyleSheet,
+} from 'react-native';
 
 class BlurView extends Component {
-  setNativeProps = nativeProps => {
+  setNativeProps = (nativeProps) => {
     if (this._root) {
       this._root.setNativeProps(nativeProps);
     }
@@ -12,7 +17,7 @@ class BlurView extends Component {
   render() {
     return (
       <NativeBlurView
-        ref={e => (this._root = e)}
+        ref={(e) => (this._root = e)}
         {...this.props}
         style={StyleSheet.compose(styles.transparent, this.props.style)}
       />
@@ -21,7 +26,7 @@ class BlurView extends Component {
 }
 
 const styles = StyleSheet.create({
-  transparent: { backgroundColor: 'transparent' },
+  transparent: {backgroundColor: 'transparent'},
 });
 
 BlurView.propTypes = {
